internal/resolvers/youtube/load.go

```diff
@@ -68,3 +68,92 @@ func load(videoID string, r *http.Request) (interface{}, time.Duration, error) {
 		Thumbnail: thumbnail,
 	}, cache.NoSpecialDur, nil
 }
+
+func loadChannels(channelCacheKey string, r *http.Request) (interface{}, time.Duration, error) {
+	youtubeChannelParts := []string{
+		"statistics",
+		"snippet",
+	}
+
+	log.Println("[YouTube] GET channel", channelCacheKey)
+	builtRequest := youtubeClient.Channels.List(youtubeChannelParts)
+
+	channelId := deconstructChannelIdFromCacheKey(channelCacheKey)
+	if channelId.channelType == CustomChannel {
```

Comment:
Pretty sure we should use `channelID` as the variable name here. Same thing with all the other `Id` in variable names - those should be changed to `ID` instead.

Reply:
Will adjust, C# habits